jobs/datadog-bbs/templates/bbs_check.yaml.erb

```diff
@@ -0,0 +1,17 @@
+init_config:
+  default_timeout: 5
+instances:
+  - name: BBS
+    url: https://<%= p('bbs.url')%>/v1/actual_lrp_groups/list
```

Comment:
I've realised after merging that there's a bug here. It works OK in alphagov/paas-cf#818 where we pass `bbs.service.cf.internal:8889`, but if you use the spec default you'll get an invalid URL of `https://https://localhost:8889/v1/actual_lrp_groups/list`.

We should either rename the property to `bbs.hostname` and remove the protocol from the default value, or remove the protocol from template and pass it from `alphagov/paas-cf`.